modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyDetail/index.tsx

```diff
@@ -14,19 +15,52 @@
 
 import './index.scss';
 
-import {useState} from 'react';
+import {useEffect, useState} from 'react';
 
 import NavigationBar from '../../../../../common/components/navigation-bar';
 import {getWebDavUrl} from '../../../../../common/utils/webdav';
 
-const PolicyDetail = () => {
+type PolicyDetails = {
+	annualMileage: number;
+	creditRating: string;
+	features: string;
+	firstName: string;
+	gender: string;
+	highestEducation: string;
+	maritalStatus: string;
+	model: string;
+	occupation: string;
+	ownership: string;
+	primaryUsage: string;
+};
+
+const PolicyDetail = ({application}: any) => {
```

Comment:
Please, advise the correct type for it.
```suggestion
const PolicyDetail = ({application}: ???) => {
```

Reply:
This object is already prefilled as any in the prop, should I define a type for it anyway?

Reply:
Done!